### DISCUSSION TITLE
How did you enabled the new GitHub issues template?

### DISCUSSION THREAD

**Discussion by derech1e** [Tyrrrz/YoutubeDownloader#216] [Q&A] [answered]
Hi, I saw that you have already activated the new beta of Github issues. However, when I want to use it on my own, it doesn't quite work. 
I used exactly the same config files as in your repository.
How did you get it to work?

**Comment by Tyrrrz** [comment, marked as answer]
I applied for private alpha here: https://docs.google.com/forms/d/e/1FAIpQLSea1QJ_9Os2gWpGCtpESjeV6VvghKn0iELvbYo8iQFi9GyFsA/closedform

But it's closed now.
